src/LandingPage.tsx

```diff
@@ -86,7 +86,7 @@ const LandingPage: React.FC = () => {
       },
       body: `idtoken=${onlineResponse.tokenId}`,
     };
-    return fetch(`http://paralibrary.digital/api/login`, options)
+    return fetch(`http://paralibrary.digital/api/auth/login`, options)
```

Comment:
This route should be changing as part of https://github.com/ParaLibrary/paralibrary-backend/pull/21. I can do these in parallel once both are ready to go